exercises/input-output/README.md

```diff
@@ -12,22 +12,49 @@ For the exercises below, we've provided the starter project above.
 
 ## :pushpin: File Reading
 
-Write a program that uses a `FileReader` to read the content of a file, and print it to the screen, line by line.
+Create a class called `FileManager` with the following methods:
 
-We've provided the input `exercise.txt` file in this project's resources folder.
+```java
+public BufferedReader getReader(String filename)
+```
+
+The `getReader` method should:
+- return a `BufferedReader` object for the provided file
 
 ## :pushpin: File Writing
 
-Expand on the previous program, and use a `FileWriter` to copy the content of the `exercise.txt` to a new file.
+Add the following method to the `FileManager` class:
+
+```java
+public void copy(String inputFile, String outputFile)
+```
+
+The `copy` method should:
+- copy the contents of a text file at the path defined by `inputFile` to the path defined by `outputFile`.
+- create `outputFile` if it doesn't exist and overwrite it, if it does
 
-The output file should be created in this project's resources folder.
+### :bookmark: Note:
+The paths to `inputFile` and `outputFile` are relative to the root directory of this repository (e.g. `exercises/input-output/src/main/resources/exercise.txt`)
 
 ## :pushpin: `java.nio.*`
 
-Using the `java.nio.*` classes, improve on the previous program.
+Using the `java.nio.*` classes, improve the `FileManager` class:
 
 ## :pushpin: File Content Handling
```

Comment:
Either remove the `:` at the end of line 41 or make line 43 a subheading of the `java.nio.*` section. It's reading as though `## :pushpin: File Content Handling` is a subtask of the `java.nio.*` task

Reply:
Updated content